packages/cli/src/index.ts

```diff
@@ -0,0 +1,12 @@
+import * as yargs from 'yargs';
+
+import mockCommand from './commands/mock';
+
+const _v = yargs
+  .scriptName('prism')
+  .version(false)
```

Comment:
Does this mean prism --version will not be an option? 

Currently we have: 

```prism --version
@stoplight/prism-cli/3.0.0-alpha.1 darwin-x64 node-v10.15.3```

(ha im on a very old version).

Reply:
Very good catch. Fixed in 982af0c